improver_tests/acceptance/SHA256SUMS

```diff
@@ -754,8 +758,10 @@ ebac866c30e11898a4925b070d79077314ddf6f6685dfef5761e7f21a872bf89  ./phase-probab
 ed7a7806032f75669575cc37941d2d7570ebfcfe4784028a11a71ee72df35f87  ./phase-probability/spot/snow_sleet_percentiles.nc
 ae048c636992e80b79c6cbb44b36339b30ea8d0ef1db72cd3f4de8766346fa1d  ./recursive-filter/input.nc
 b6cdb8bf877bb0b3b78ad224b50b9272b65732bf9e39a88df704209e228bf4c0  ./recursive-filter/input_masked.nc
+d04127a1f5256333b7cffefe1a9966758ab00a580ec347e248d85679c3f01799  ./recursive-filter/input_variable_masked.nc
 dfa0e2482041db9fd662e566063d470a25b6491d2b708b13873a6f93bed2d6ad  ./recursive-filter/kgo_basic.nc
 cb87460b75f92a775d850a2eee2b8b89e63bcf2dad15f84b5dcea5bc936fd22b  ./recursive-filter/kgo_internal_mask_with_re_mask.nc
+d084d978b1253134d195cd53c28f146f35956a843a0f5c56ac5410386c699be5  ./recursive-filter/kgo_variable_internal_mask_with_re_mask.nc
```

Comment:
Is there a side-by-side test to show the variable_mask functionality? It looks like `kgo_internal_mask_with_re_mask.nc` only has one threshold whilst `kgo_variable_internal_mask_with_re_mask.nc` has two. It might be useful to update `kgo_internal_mask_with_re_mask.nc`, so that it's easiest to see the impact of this behaviour.

Reply:
There's no side-by-side test as I think the use cases for allowing/dis-allowing non-matching masks are a bit different. For example, if you knew that you were applying a land-sea mask to your data then this should match across every spatial slice. On the other hand, my use-case is applying a mask to zero values, which may well differ in each spatial slice as the forecast changes across time or thresholds.

I'm not sure what a good comparison would look like, if one is desirable?

Reply:
OK, I think that's reasonable, so I don't think a side-by-side test is necessary.